TaskManager/Modules/Add Task/View/AddTaskViewController.swift

```diff
@@ -73,7 +74,7 @@ class AddTaskViewController: UIViewController, AddTaskView {
     }
     
     @IBAction func createTask(_ sender: Any) {
-        presenter.createTask(title: titleTextField.text ?? "", description: subtitleTextField.text)
+        presenter.createTask(title: titleTextField.text ?? "", subtitle: subtitleTextField.text, isActive: true, uuid: UUID())
```

Comment:
usually UUID() is called in Presenter because its part of presenter to generate new uuid